packages/ag-charts-community/src/module/optionsGraph.ts

```diff
@@ -231,6 +230,29 @@ export class OptionsGraph extends AdjacencyListGraph<unknown, string> implements
         return vertex;
     }
 
+    hasUserOption(path: Array<string>) {
+        return hasPathSafe(this.userOptions, path);
+    }
+
+    hasThemeOverride(path: Array<string>) {
```

Comment:
Micro-optimisation: How often is this being invoked? Might be worth making the constant paths here declared statically if so?

Reply:
It's not in this PR. I believe it will be used in the highlight defaults, so once or twice per series.